src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +391,46 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Prepare shown languages, source all available languages from the addons themselves
+	std::vector<std::string> languages_available;
+	for(const auto& a : addons_) {
+		for (const auto& b : a.second.locales) {
+			languages_available.push_back(b);
+		}
+	}
+	std::sort(languages_available.begin(), languages_available.end());
+	auto last = std::unique(languages_available.begin(),languages_available.end());
+	languages_available.erase(last, languages_available.end());
+
+	// Erase pt value, since otherwise Portugues do Brasil will be shown twice in the language list
+	languages_available.erase(std::remove(languages_available.begin(), languages_available.end(), "pt"), languages_available.end());
+
+	for (long unsigned int i = 0; i < languages_available.size(); i++) {
+		std::string myLangCodeString = langcode_to_string(languages_available[i]);
+		// Only show languages, which have a translation as per langcode_to_string() method
+		// Do not show tranlations with their langcode e.g. "sv_SV"
+		if (myLangCodeString != "")
+			language_filter_types_.emplace_back(std::pair<int, std::string>(i, langcode_to_string(languages_available[i])));
+		// Remember en_GB pos in toggle vector, so packages with Translations: None can be defaulted to English (GB) lang_string
+		if (languages_available[i] == "en_GB")
+			en_GB_toggle_position = language_filter_types_.size() - 1;
```

Comment:
Why is en_GB being treated as a special case?

Reply:
All of the addons, where no translations are specified are assumed to have at least an English translation, so defaulting to English, that's why I need to know en_GBs position in the scraped lang code vector. 

As for why it is en_GB and not en_US, it's personal preference, I reckon

Reply:
Right, but why is this needed in the first place? Showing everything when no language filter options are chosen should be enough by itself, I would think.

Reply:
Ah, now I see what you mean. Indeed, a person knowing that every addon has an English translation would then simply not filter the list. But then the English filter option would be effectively filtering nothing, if every addon really has an English translation and we would assume that every person using the addon store knows that. 

So how I see it we should either remove en_GB/en_US language filter option completely, as every addon is English by default anyways or we should keep it for UX purposes and show everything when it is selected, making it so that it filters nothing.

Reply:
I'm not really following. There should be no assumption that every add-on is English or any particular language. Leaving the language filter empty simply means that the player doesn't care what translations are available.

If there exists an add-on with an en_US or en_GB translation added, then it should end up in the language filter drop down the same as any other language.

Reply:
To add to that - the default language for any particular add-on is simply unknown. All we have available to be able to possibly filter on is what additional translations the add-on author makes available:
* If the default text for an add-on is in **English** and they add a **French** translation, then **French** is what it is filterable on.
* If the default text for an add-on is in **French** and they add an **English** translation, then **English** is what it is filterable on.
* When the language filter is empty, both add-ons from those two above examples should be displayed, not because we're assuming all add-ons are English by default but because the available translations aren't even looked at in the first place when deciding what to display on the UI.

Reply:
Shouldn't the default language of an add-on be known and included? That would make a lot more sense to me. How are people supposed to find add-ons in their language if some of them might have that language as the default and others might have it as a translation?

Reply:
Perhaps, but I'd say that's something for a separate issue/PR.